calico_containers/tests/st/test_base.py

```diff
@@ -44,3 +48,32 @@ def start_etcd(self):
             initial_cluster="calico=http://%s:2380" % self.ip,
             initial_cluster_state="new",
         )
+
+    def assert_powerstrip_up(self, host):
```

Comment:
I don't get how this asserts that powerstrip is up.  It looks like it just tries `docker ps` until it gets a successful error code, which will happen regardless of whether Powerstrip is up.